libafl_qemu/src/calls.rs

```diff
@@ -186,7 +186,13 @@ where
                 }
             }
 
-            iaddr += insn.bytes().len() as u64;
+            #[cfg(cpu_target = "arm")]
+            let insn_len = insn.bytes().len() as u32;
+            #[cfg(not(cpu_target = "arm"))]
+            let insn_len = insn.bytes().len() as u64;
```

Comment:
here too, and in all the other places, remove the cfg arm and use GuestAddr as integer type instead of u64/u32